bokehjs/src/lib/models/glyphs/line.ts

```diff
@@ -32,9 +32,10 @@ export class LineView extends XYGlyphView {
 
   protected _render(ctx: Context2d, indices: number[], data?: LineData): void {
     const {sx, sy} = data ?? this
+    const nonselection = this.parent.nonselection_glyph == this
 
-    let last_i: number | null = null
-    const gap = (i: number) => last_i != null && i - last_i != 1
+    let iprev: number | null = null
```

Comment:
I have changed the name from `last_i` to `iprev` to agree with the webgl implementation and because I think "prev" is a more appropriate description than "last" here. But it means there are more line changes than there would be otherwise, so I can revert the name change if that is preferred.